src/Ng/CommandHelpers.cs

```diff
@@ -29,13 +29,20 @@ namespace Ng
 {
     public static class CommandHelpers
     {
-        public static IDictionary<string, string> GetArguments(string[] args, int start)
+        private static readonly int DefaultKeyVaultSecretCachingTimeout = 60 * 60 * 6; // 6 hours;
+        private static readonly HashSet<string> NotInjectedKeys = new HashSet<string>(StringComparer.OrdinalIgnoreCase) {
```

Comment:
nit: curly on the next line

Reply:
Also it looks like line 35, `};`, has one too many indentations.